splink/linker.py

```diff
@@ -2178,6 +2183,94 @@ def _compute_metrics_nodes(
 
         return df_node_metrics
 
+    def _compute_metrics_edges(
+        self,
+        df_node_metrics: SplinkDataFrame,
+        df_predict: SplinkDataFrame,
+        df_clustered: SplinkDataFrame,
+        threshold_match_probability: float,
+    ) -> SplinkDataFrame:
+        """
+        Internal function for computing edge-level metrics.
```

Comment:
This logic could possibly be moved to a file like edge_metrics.py to reduce the size of linker.py?